src/main/java/io/github/aquerr/futrzakbot/discord/audio/FutrzakAudioPlayer.java

```diff
@@ -290,4 +290,8 @@ public void stopCurrentTrack()
     {
         this.audioPlayer.stopTrack();
     }
+
+    public void jumpTo(long position) {
```

Comment:
Można byłoby zamienić nazwę zmiennej na `positionSeconds`, będzie od razu wiadomo że są to sekundy.